definitions/npm/mersenne-twister_v1.x.x/flow_v0.33.x-/mersenne-twister_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'mersenne-twister' {
+  declare class $npm$MersenneTwister {
```

Comment:
Oh, I don't know about `declare var exports`, that's worth a try in the future. I've seen a few other modules prefix things (though leave them in the global scope) instead of leave them regular-named inside the module, so I don't think this is too inconsistent with what else is here. I think Flow is sorely lacking a real way to mark things private in a definition.